iceoryx_examples/icedelivery/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ target_compile_options(iox-cpp-publisher PRIVATE ${ICEORYX_WARNINGS} ${ICEORYX_S
 
```

Comment:
Traceability. What does this have to do with the component diagram? Please create a separate issue and adjust the commit message iox number to be traceable.

Reply:
This is an example motivating an extra `runtime` library and belongs to the diagram.